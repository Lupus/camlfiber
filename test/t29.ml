[@@@warning "-52"]

let print_exc e = print_string (Printexc.to_string e);
                  print_newline ()

let f = Fiber.create (fun () ->
            Fiber.sleep 0.01;
            "ok\n") ()
let _ =
  try print_string (Fiber.join f)
  with Invalid_argument("Fiber.yield") as e -> print_exc e (* FIXME: should it be "Fiber.join" ? *)
