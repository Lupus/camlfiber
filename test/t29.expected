Invalid_argument("Fiber.yield")
